### ISSUE TITLE
Make mission control better to watch generic roll-outs with

### ISSUE THREAD

**Issue by fbertsch** [mozilla/missioncontrol#183]
For generic roll-outs, we probably don't need to worry about crashes, but more about things like:
- total number of clients
- total number of downloads
- session hours

In addition, the plots should auto update without refreshing.

**Comment by fbertsch** [comment]
This will also require reading from new streaming data sources.

**Comment by wlach** [comment]
Yup, definitely. We actually track client counts and session hours as part of the error aggregates already, but maybe those numbers would be better off in their own series.
